backend/controllers/process/process.js

```diff
@@ -87,7 +112,23 @@ const editProcess = async (req, res) => {
         return res.status(422).json({ errors: errors.array()[0] });
     }
     if(auth.auth()){
-        return res.status(404).json({ message: "Could not find the given process" });
+        try{
+            const result = await prismaClient.process.update({
+                where: {
+                    id: parseInt(req.body.id)
+                },
+                data: {
+                    type: req.body.type ? parseInt(req.body.type) : undefined,
```

Comment:
Ich weiß nicht, ob das so gewollt ist, aber wenn man hier nur eine Sache, oder gar keine angibt(type oder official), dann wird das, was man nicht mitgibt, mit undefined überschrieben. Es gehen da also Daten verloren. Und ist nicht type sowieso not null? Kann man das überhaupt auf undefined setzen?